test/unittests/iotools/converters/adni_to_bids/test_adni_utils.py

```diff
@@ -339,3 +339,68 @@ def test_get_closest_visit(closest_visit_timepoints, image_acquisition_date, exp
         ),
         closest_visit_timepoints[expected],
     )
+
+
+@pytest.fixture
+def input_df():
+    return pd.DataFrame(
+        {
+            "foo": ["foo1", "foo2", "foo3"],
+            "bar": [1, 2, 3],
+            "baz": [True, False, False],
+            "foobar": [4, 5, 6],
+        }
+    )
+
+
+@pytest.mark.parametrize(
+    "csv_name,csv_to_look_for",
+    [("adnimerge.csv", "adnimerge"), ("adnimerge_20Oct2023.csv", "adnimerge")],
+)
+def test_load_clinical_csv(tmp_path, input_df, csv_name, csv_to_look_for):
+    from clinica.iotools.converters.adni_to_bids.adni_utils import load_clinical_csv
+
+    input_df.to_csv(tmp_path / csv_name, index=False)
+    assert_frame_equal(load_clinical_csv(tmp_path, csv_to_look_for), input_df)
+
+
+@pytest.mark.parametrize(
```

Comment:
I would remove the parametrization if you have only one value.

remove `csv_to_look_for` from the arguments of the test function and simply use "adnimerge" in the test function's body.